Code/Framework/AzCore/AzCore/Math/Crc.h

```diff
@@ -80,13 +80,16 @@ namespace AZ
          * Calculates the value from a block of raw data.
          */
         Crc32(const void* data, size_t size, bool forceLowerCase = false);
-        constexpr Crc32(const uint8_t* data, size_t size, bool forceLowerCase = false);
-        constexpr Crc32(const char* data, size_t size, bool forceLowerCase = false);
+        template<class ByteType, class = AZStd::enable_if_t<sizeof(ByteType) == 1>>
+        constexpr Crc32(const ByteType* data, size_t size, bool forceLowerCase = false);
+        constexpr Crc32(AZStd::span<const AZStd::byte> inputSpan);
```

Comment:
Is it possible to templatize this for span as well? 
```suggestion
        template<class ByteType, class = AZStd::enable_if_t<sizeof(ByteType) == 1>>
        constexpr Crc32(AZStd::span<const ByteType> inputSpan);
```

Reply:
Yes the span could be templatized as well, however the code is using a convention that passing text is `AZStd::string_view` and passing in binary is `AZStd::span<const AZStd::byte>`

Since the Crc32 class has both, that covers the text and binary cases